packages/immutable-rootfs/30cos-immutable-rootfs/module-setup.sh

```diff
@@ -21,7 +21,7 @@ install() {
     declare moddir=${moddir}
     declare systemdutildir=${systemdutildir}
     inst_multiple \
-        mount mountpoint yip cos-setup sort
+        mount mountpoint yip cos-setup sort rmdir
```

Comment:
I think this is some sort of leftover, as I don't see rmdir being used anywhere.

Reply:
true, but it actually failed otherwise while finding the images after install during the isoscan step. It complained about missing rmdir and prevented to boot